Spanner/src/Connection/Grpc.php

```diff
@@ -1194,6 +1156,40 @@ public function commit(array $args)
         ]);
     }
 
+    /**
+     * @param array $args
+     * @return \Generator
+     */
+    public function batchWrite(array $args)
```

Comment:
I ran unit tests and inserted debugging code in this function, and it was never executed. We need at least some unit testing to cover this function. 

Additionally the `BatchWriteTest` tests below do not run against the emulator, so AFAICT, there is no active test coverage for this feature.

Reply:
Tests added in https://github.com/googleapis/google-cloud-php/pull/7463